randomGuess.js

```diff
@@ -0,0 +1,95 @@
+const readline = require('readline');
+
+const rl = readline.createInterface(process.stdin, process.stdout);
+
+function ask(questionText) {
+	return new Promise((resolve, reject) => {
+		rl.question(questionText, resolve);
+	});
+}
+
+async function guessFcn(maxGuess, minGuess) {
+  //funciton takes the changing min and max and makes a random guess within the parameters
+	const maxNumber = Number.parseFloat(maxGuess);
```

Comment:
You can omit the Number before the parseFloat() calls. Javascript will know to use the parseFloat from the Number object automatically. Also, you may want to use parseInt() instead of parseFloat() so that you don't get any weird decimals in your program.